sdk/communication/azure-communication-callingserver/README.md

```diff
@@ -0,0 +1,215 @@
+# Azure Communication CallingServer Service client library for Java
+
+This package contains a Java SDK for Azure Communication CallingServer Service.
+
+[Source code][source] | [Package (Maven)][package] | [API reference documentation][api_documentation]
+| [Product documentation][product_docs]
+
+## Getting started
+
+### Prerequisites
+
+- An Azure account with an active subscription. [Create an account for free](https://azure.microsoft.com/free/?WT.mc_id=A261C142F).
+- [Java Development Kit (JDK)](https://docs.microsoft.com/java/azure/jdk/?view=azure-java-stable) version 8 or above.
+- [Apache Maven](https://maven.apache.org/download.cgi).
+- A deployed Communication Services resource. You can use the [Azure Portal](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource?tabs=windows&pivots=platform-azp) or the [Azure PowerShell](https://docs.microsoft.com/powershell/module/az.communication/new-azcommunicationservice) to set it up.
+
+### Include the package
+
+[//]: # ({x-version-update-start;com.azure:azure-communication-callingserver;current})
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-communication-callingserver</artifactId>
+    <version>1.0.0-preview.1</version>
```

Comment:
This version should be `1.0.0-beta.1` right?

Reply:
updated